nautobot/extras/models/jobs.py

```diff
@@ -60,6 +61,14 @@
 # objects being created within transaction.atomic().
 JOB_LOGS = "job_logs"
 
+# The JOB_RESULT_METRIC variable is a counter metric that counts executions of jobs,
+# including information beyond what a tool like flower could get by introspecting
+# the celery task queue. This is accomplished by looking one abstraction deeper into
+# the job model of Nautobot.
+JOB_RESULT_METRIC = Histogram(
+    "nautobot_job_duration_seconds", "Results of Nautobot jobs.", ["module", "name", "status"]
```

Comment:
`nautobot_job_duration_seconds` here but `nautobot_job_results_total` in the changelog?

Reply:
Good catch, I changed the metric name after adding the changelog and forgot to reflect that.